### ISSUE TITLE
1.16.x: 1.1 - Use configs folder instead of mods folder

### ISSUE THREAD

**Issue by ChefMC** [PG85/OpenTerrainGenerator#182]
Please, change OTG worlds configurations directory from `mods` to `config` folder, because I use protected launcher with mods restrictions, so any changes of files in `mods` folder will not apply and also Minecraft will close.

Or make both folders usage. Eg:

```
if (OTG_worlds_folder_exists_in_mods_folder) use_it();
else use_config_folder();
```

**Comment by ChefMC** [comment]
My players can't join server about this bug

**Comment by PG85** [comment]
Will be included for 1.16.x 0.0.1, release date TBD. Thanks!
